src/test/resources/capabilities-bstack-local-parallel-browsers.yml

```diff
@@ -0,0 +1,55 @@
+testEndpoint: http://localhost:3000
+# Specify the EndPoint of application under test above
+driverType: remoteDriver
+
+localDriver:
```

Comment:
LocalDriver shouldn't be there for this profile?

Reply:
Removed Local Driver